packages/ui-components/src/components/LoadingContainer.js

```diff
@@ -70,44 +70,47 @@ ErrorScreen.propTypes = {
   errorMessage: PropTypes.string.isRequired,
 };
 
+const Wrapper = ({ children: loadingScreenChildren, loadingContainerchildren }) => (
+  <Container className="loading-container">
+    {loadingContainerchildren}
+    <LoadingScreen className="loading-screen">{loadingScreenChildren}</LoadingScreen>
+  </Container>
```

Comment:
Add className in loadingScreen and Container so that we can modify css on top in `LoadingContainer`'s parent